cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -0,0 +1,86 @@
+#
+# Modifications, Copyright (C) 2022 Intel Corporation
+#
+# This software and the related documents are Intel copyrighted materials, and
+# your use of them is governed by the express license under which they were
+# provided to you ("License"). Unless the License provides otherwise, you may not
+# use, modify, copy, publish, distribute, disclose or transmit this software or
+# the related documents without Intel's prior written permission.
```

Comment:
Sorry, what License this file is referring to? cc: @zpao

Reply:
This looks like potentially an internal header and not something that should get included here.